src/utils/env.ts

```diff
@@ -79,6 +79,10 @@ export const ENV = {
             .asString(),
     },
 
+    INSTITUTIONS_PSP_CHANNEL_TYPES: getConfig("REACT_APP_INSTITUTIONS_PSP_CHANNEL_TYPES", {default: "PSP", type: Type.Array} ),
```

Comment:
centralisation is a great idea.

Do you think it's a better solution create a constant file (for this and other constants) and using `ENV` only for variable?

Reply:
Moved to the constants file